src/pico/i_video.c

```diff
@@ -1060,48 +902,23 @@ void __scratch_x("scanlines") fill_scanlines() {
 }
 #pragma GCC pop_options
 
-#if PICO_ON_DEVICE
-#define LOW_PRIO_IRQ 31
-#include "hardware/irq.h"
-
-static void __not_in_flash_func(free_buffer_callback)() {
-//    irq_set_pending(LOW_PRIO_IRQ);
-    // ^ is in flash by default
-    *((io_rw_32 *) (PPB_BASE + M0PLUS_NVIC_ISPR_OFFSET)) = 1u << LOW_PRIO_IRQ;
```

Comment:
anything to do with irq is the interrupt code that would start the next VGA render cycle, so it can all get axed